bogota-july-2018/todo-react-list/src/Components/InputForm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export default class InputForm extends React.Component {
+    constructor() {
```

Comment:
You don't need to write the constructor if you're not going to write anything there